src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Don't use * selector (it impacts performance), better use ``html`` or ``body`` tag.

![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/d6ded0d1-0211-40db-83a5-75d0f84541d5)